src/foam/nanos/script/Script.js

```diff
@@ -74,9 +79,14 @@ foam.CLASS({
       value: true
     },
     {
-      class: 'Boolean',
-      name: 'scheduled',
-      hidden: true
+      class: 'foam.core.Enum',
+      of: 'foam.nanos.script.ScriptStatus', 
+      name: 'status',
+      visibility: foam.u2.Visibility.RO,
+      factory: function()
```

Comment:
Please follow the [Google's JS style conventions](https://google.github.io/styleguide/jsguide.html#formatting-braces) for this and other places.